pages/user-interfaces.html

```diff
@@ -36,15 +25,14 @@
 can be extended by the minor Coq
 mode <a href="https://github.com/cpitclaudel/company-coq">Company-Coq</a>.
 Proof General was one of the first user interface for proof
-assistants, and is one of the most used user interface for Coq today.
-Proof General and Company-Coq give access to many productivity
-shortcuts, including auto-completion and documentation.  On the other
-hand, async proof processing is not supported.
+assistants. See e.g.
+<a href="https://proofgeneral.github.io/doc/master/userman/Coq-Proof-General/">here</a>
+for an excerpt of Coq-specific features.
 </li>
 
 <li>
-<strong>Vim/NeoVim</strong> users can use the actively
-maintained <a href="https://github.com/whonore/Coqtail">Coqtail</a>
+<strong>Vim/NeoVim</strong> users can use the
+<a href="https://github.com/whonore/Coqtail">Coqtail</a>
 extension.  This extension is partly based upon a previous, and now
 unmaintained, Vim extension
 called <a href="https://github.com/the-lambda-church/coquille">Coquille</a>.
```

Comment:
IMHO time to remove this info from here too.

Reply:
Agreed.